util/src/main/java/com/liferay/faces/util/render/internal/RenderKitImpl.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-2015 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.util.render.internal;
+
+import javax.faces.render.RenderKit;
+import javax.faces.render.RenderKitWrapper;
+import javax.faces.render.Renderer;
+
+
+/**
+ * @author  Kyle Stiemann
+ */
+public class RenderKitImpl extends RenderKitWrapper {
+
+	// Private Members
+	private RenderKit wrappedRenderKit;
+
+	public RenderKitImpl(RenderKit wrappedRenderKit) {
+		this.wrappedRenderKit = wrappedRenderKit;
+	}
+
+	@Override
+	public Renderer getRenderer(String family, String rendererType) {
+
+		Renderer renderer = super.getRenderer(family, rendererType);
+
+		if ("javax.faces.Body".equals(rendererType)) {
+			renderer = new BodyRendererImpl(renderer);
```

Comment:
@ngriffin7a,
I think that we may be able to do something like this in the Bridge Impl. Basically, our `BodyRendererBridgeImpl` would wrap other body renderers and do any bridge stuff, then delegate. This would remove the need for the `BodyRendererPrimefacesImpl` since we would call through to the PF BodyRenderer (supplying a `ResponseWriter` which changes `body` tags to `div` tags of course). That would make it much easier to fix https://issues.liferay.com/browse/FACES-2493, since we could pass a different response writer instead. After you merge this code, I can test whether my idea will work and send you a pull if you think that's a good idea.